aws-android-sdk-s3-test/src/androidTest/java/com/amazonaws/services/s3/AclIntegrationTest.java

```diff
@@ -124,35 +125,35 @@ public void testCreateBucketWithAcl() throws Exception {
      */
     @Test
     public void testCannedAcls() throws Exception {
-        s3.createBucket(bucketName);
-        S3IntegrationTestBase.waitForBucketCreation(bucketName);
+        s3.createBucket(BUCKET_NAME);
+        S3IntegrationTestBase.waitForBucketCreation(BUCKET_NAME);
 
-        s3.putObject(bucketName, key,
+        s3.putObject(BUCKET_NAME, KEY,
                 new ByteArrayInputStream("foobarbazbar".getBytes(StringUtils.UTF8)),
                 new ObjectMetadata());
-        final Owner bucketOwner = s3.getBucketAcl(bucketName).getOwner();
+        final Owner bucketOwner = s3.getBucketAcl(BUCKET_NAME).getOwner();
 
         // Public Read Canned ACL
-        s3.setBucketAcl(bucketName, CannedAccessControlList.PublicRead);
-        s3.setObjectAcl(bucketName, key, CannedAccessControlList.PublicRead);
-        AccessControlList bucketAcl = s3.getBucketAcl(bucketName);
-        AccessControlList objectAcl = s3.getObjectAcl(bucketName, key);
+        s3.setObjectAcl(BUCKET_NAME, KEY, CannedAccessControlList.PublicRead);
+        AccessControlList bucketAcl = s3.getBucketAcl(BUCKET_NAME);
+        AccessControlList objectAcl = s3.getObjectAcl(BUCKET_NAME, KEY);
 
         AccessControlList[] accessControls = new AccessControlList[] {
```

Comment:
What is this used for? I see the variable being overridden below, is this necessary, or should we just move the declration to line 157?